### PR TITLE
[ACCUMULO-4775] Skip negate column entirely in ColumnAgeOffFilter

### PR DESCRIPTION
**Pull request by adamjshook**

### CODE REVIEW

File: core/src/main/java/org/apache/accumulo/core/iterators/user/ColumnAgeOffFilter.java [hunk 1]
```diff
@@ -44,8 +44,10 @@ public TTLSet(Map<String,String> objectStrings) {
         String column = entry.getKey();
         String ttl = entry.getValue();
```
Comment (keith-turner) [inline core/src/main/java/org/apache/accumulo/core/iterators/user/ColumnAgeOffFilter.java]: I was thinking about the case of ignoring only whitespace also. Could trim here like the following to do that.

```java
String ttl  = entry.getValue().trim();
```

Comment (adamjshook) [reply core/src/main/java/org/apache/accumulo/core/iterators/user/ColumnAgeOffFilter.java]: Good idea!  Pushed the `trim`.

File: shell/src/test/java/org/apache/accumulo/shell/commands/SetIterCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    Field field = reader.getClass().getSuperclass().getDeclaredField("out");
+    field.setAccessible(true);
+    field.set(reader, out);
+
+    // Parsing iterator options
+    reader.flush();
+    EasyMock.expectLastCall().times(3);
+
+    EasyMock.expect(shellState.getReader()).andReturn(reader);
+    EasyMock.expect(reader.readLine(EasyMock.anyObject(String.class))).andReturn(""); // Default of 'false'
```
Comment (keith-turner) [inline shell/src/test/java/org/apache/accumulo/shell/commands/SetIterCommandTest.java]: Is this when the shell ask for the negate option?  If so would be nice to comment for each of these line what the shell is asking for.

Comment (adamjshook) [reply shell/src/test/java/org/apache/accumulo/shell/commands/SetIterCommandTest.java]: Yup -- pushed some comments.

**Comment by keith-turner** [comment]
I am going merge this

**Pull request merged**